backend/app/Models/TemplateStepMedia.php

```
@@ -0,0 +1,77 @@
+    protected function casts(): array
+    {
+        return [
+            'file_size' => 'integer',
+            'sort_order' => 'integer',
+        ];
+    }
+
+    protected static function booted(): void
+    {
+        // Only drop the file on a permanent delete - a soft delete is reversible.
+        static::forceDeleted(function (self $media) {
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\TemplateStepMedia::forceDeleted().


(staticMethod.notFound)